docs/SIL.rst

```diff
@@ -705,6 +715,20 @@ generic constraints:
   * Runtime-sized types
   * Non-class protocol types
   * @weak types
+  * Types that can't satisfy the requirements for being loadable because they
+    care about the exact location of their value in memory and need to run some
+    user-written code when they are copied or moved. For example:
+
+    * Addresses of values of Swift ``@weak`` types are registered in a global
+      table. That table needs to be adjusted when a ``@weak`` value is copied
+      or moved to a new address.
+
+    * A non-COW collection type with a heap allocation needs to allocate memory
```

Comment:
Can you give a specific example of what you are talking about?

Reply:
Added "(like ``std::vector`` in C++)".